### ISSUE TITLE
CustomBatchNormalization error when update params

### ISSUE THREAD

**Issue by clockwiser** [rarilurelo/keras-VAE#2]
In keras 2 environment, running the training1 causes error.
It says, two parameters from custombatch normalization are not references, so immutable to update.
By the way, I changed the initilization to initializers since keras 2 provides no more support of initialization class.

Guess this class change somehow modified the weights of the custom layer to be immutable.
Any help is welcom.

**Comment by clockwiser** [comment]
initializer creates constant tensors, so is the error. Where can find the old version source of initializations.py?

**Comment by clockwiser** [comment]
After find old version, changed the custom batch normalization, yet problems there are. Later found keras BatchNormalization is based on the same theory as below.

https://arxiv.org/abs/1502.03167

So switched to keras BatchNormalization, and everything looks working.
